fighthealthinsurance/ml/ml_citations_helper.py

```
@@ -1,100 +1,277 @@
+        # Normalize inputs - trim whitespace and convert to lowercase
+
+        if denial:
+            procedure = denial.procedure.strip().lower() if denial.procedure else ""
+            diagnosis = denial.diagnosis.strip().lower() if denial.diagnosis else ""
+        else:
+            procedure = procedure_opt.strip().lower() if procedure_opt else ""
+            diagnosis = diagnosis_opt.strip().lower() if diagnosis_opt else ""
+
+        # Skip if we don't have enough information
+        if not procedure and not diagnosis:
+            logger.debug(f"Missing procedure or diagnosis for generic citations")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)